EnglishFriend.xcodeproj/project.pbxproj

```diff
@@ -779,37 +805,60 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		6145471C265B5C7F00C85B96 /* XCRemoteSwiftPackageReference "firebase-ios-sdk" */ = {
+		616B720026A92EB6005D928D /* XCRemoteSwiftPackageReference "Kingfisher" */ = {
```

Comment:
I do have some reservations about using Kingfisher again. I know you're aware of the issues we've seen with it in the past.